src/rollback_to_stable/rts_btree_walk.c

```diff
@@ -268,11 +268,11 @@ __wt_rts_btree_walk_btree_apply(
         dhandle_allocated = true;
 
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-          WT_RTS_VERB_TAG_TREE
-          "rolling back tree. modified=%s, durable_timestamp=%s > stable_timestamp=%s: %s, "
-          "has_prepared_updates=%s, txnid=%" PRIu64 " > recovery_checkpoint_snap_min=%" PRIu64
-          ": %s",
-          S2BT(session)->modified ? "true" : "false",
+          WT_RTS_VERB_TAG_TREE "rolling back tree. uri=%s, btree_id=%" PRIu32
+                               ", modified=%s, durable_timestamp=%s > stable_timestamp=%s: %s, "
+                               "has_prepared_updates=%s, txnid=%" PRIu64
+                               " > recovery_checkpoint_snap_min=%" PRIu64 ": %s",
+          uri, S2BT(session)->id, S2BT(session)->modified ? "true" : "false",
```

Comment:
Nitpick: keep `S2BT(session)` on the stack? I don't mind either way.

Reply:
The `S2BT` is used later in the function also. As the usage is increased, better to be a local variable?

Reply:
Fair enough - done in 7bc3c7f.